File: cross-application-feature-development-management/Directories/ScriptsDirectory.cs
namespace cross_application_feature_development_management.Directories
{
    public class ScriptsDirectory(
            CommandLineArgs commandLineArgs
        )
    {
        public string GetName()
        {
            var scriptsDirectoryNameKey = commandLineArgs.GetKey("ScriptsDirectoryNameKey");
            var scriptsDirectoryName = CommandLineArgs.GetByKey(scriptsDirectoryNameKey);
            return scriptsDirectoryName;
        }
    }
}

File: dotnet/cafdemvers/zzq-cafdemwimu/cafdemwimu.console/Directories/Repository/Dotnet/Scripts/ScriptsDirectory.fs
namespace cafdemwimu.console.Directories.Repository.Dotnet.Scripts

open cafdemwimu.console

type ScriptsDirectory() =
    member _.GetPath() =
        let scriptsDirectoryNameKey = "--scripts-directory"
        let scriptsDirectoryName =
            CommandLineArgs.GetByKey(scriptsDirectoryNameKey)
        scriptsDirectoryName
